explorer/interpreter/value.cpp

```diff
@@ -1167,7 +1167,7 @@ auto FindFunction(std::string_view name,
       }
       case DeclarationKind::FunctionDeclaration: {
         const auto& fun = cast<CallableDeclaration>(*member);
```

Comment:
This one specifically (and the following `FindFunction`) can simply cast to `FunctionDeclaration` and keep `.name()`, this will also clarify what we are working with.
I double checked that this one does not have a falltrough :).